core/chains/evm/client/simulated_backend_client.go

```diff
@@ -64,37 +64,66 @@ func (c *SimulatedBackendClient) checkEthCallArgs(
 			"must be an eth.CallArgs, got %+#v", args[0])
 	}
 	blockNumber, err := c.blockNumber(args[1])
-	if err != nil || blockNumber.Cmp(c.currentBlockNumber()) != 0 {
+	if err != nil {
 		return nil, nil, fmt.Errorf("fourth arg to SimulatedBackendClient.Call "+
-			"must be the string \"latest\", or a *big.Int equal to current "+
-			"blocknumber, got %#+v", args[1])
+			"must be the string \"latest\", or a *big.Int, got %#+v", args[1])
 	}
+
+	// to and from need to map to a common.Address but could come in as a string
 	ca := CallArgs{
-		From: callArgs["from"].(common.Address),
-		To:   *callArgs["to"].(*common.Address),
+		To:   interfaceToAddress(callArgs["to"]),
 		Data: callArgs["data"].(hexutil.Bytes),
 	}
+
+	// from is optional in the standard client; default to 0x when missing
+	if value, ok := callArgs["from"]; ok {
+		ca.From = interfaceToAddress(value)
+	} else {
+		ca.From = common.HexToAddress("0x")
+	}
+
 	return &ca, blockNumber, nil
```

Comment:
It will just use latest by default. This change makes the behavior of `CallContext` and `BatchCallContext` match regarding the block number passed to the simulated backend.